indico/modules/rb/controllers/user/reservations.py

```diff
@@ -24,6 +24,7 @@
 
 from indico.core.db import db
 from indico.core.errors import IndicoError, NoReportError, NotFoundError
+from indico.modules.rb import settings as rb_settings
```

Comment:
at some point we should rename that `settings` to `rb_settings` in `__init__.py` (but not in this PR please, once all the current RB changes have been merged) instead of aliasing it at import time... not sure what the hell I was thinking when going for import-time aliasing initially 🐒